internal/services/network/express_route_circuit_peering_resource.go

```diff
@@ -188,6 +210,11 @@ func resourceExpressRouteCircuitPeering() *pluginsdk.Resource {
 				Optional:     true,
 				ValidateFunc: azure.ValidateResourceID,
 			},
+
+			"gateway_manage_etag": {
```

Comment:
i don't see this in the docs? also shouldn't it be
```suggestion
			"gateway_manager_etag": {
```